reports/3_generate_report_emails.py

```diff
@@ -0,0 +1,144 @@
+from calitp.tables import tbl
+from siuba import *
+import sys
+from postmarker.core import PostmarkClient
+import os
+import pandas as pd
+
+# +
```

Comment:
Please add a full sentence wherever these placeholder comments are to better describe what is happening in the code.

Reply:
These comments are from a tool called jupytext, which allows people to open scripts at jupyter notebooks (and edit them on e.g. jupyterhub)!

https://github.com/mwouts/jupytext/blob/main/docs/formats.md#The-light-format